uber/templates/preregistration/preregbase.html

```diff
@@ -64,12 +66,12 @@
             row: '#badge-types',
             selector: '.badge-type-selector',
             options: [{
```

Comment:
Ribbon_and_or_badge displays exactly what it says on their ribbon - so Volunteers would see "Volunteer," Dealers would see "Dealer," etc.